muhireheir/mybrand-1
# Ft login express

> #### What does this PR do?

> * validate login  form
> * log user in
> #### Description of Task to be completed
> * checks if user entered correct data format and if the form is not empty
> #### How should this be manually tested?
>  * clone the repository
> * change to ft-login-express
> * go into server folder
> * open  terminal  in server folder
> * run  the  following   codes `npm start`
> * open postman
> * i  url enter the following adress `localhost:300/signin`
> * send request with post method
> * see  the image below  to have an idea of params must be in the body of the [request](url)
> * Done.
> #### Any background context you want to provide?
> * allowed user with account to lig in
> #### What are the relevant Trello story?
>  * https://trello.com/c/8NhIhNzK
> #### Screenshots (if appropriate)
![logi](https://user-images.githubusercontent.com/26524106/93249074-0a19c100-f791-11ea-805b-b41219386abd.png)
